data/com.usebottles.bottles.appdata.xml.in

```diff
@@ -60,38 +60,21 @@
             <image>https://raw.githubusercontent.com/bottlesdevs/Bottles/master/data/appstream/5.png</image>
         </screenshot>
     </screenshots>
-    <content_rating type="oars-1.1">
-        <content_attribute id="violence-cartoon">none</content_attribute>
-        <content_attribute id="violence-fantasy">none</content_attribute>
-        <content_attribute id="violence-realistic">none</content_attribute>
-        <content_attribute id="violence-bloodshed">none</content_attribute>
-        <content_attribute id="violence-sexual">none</content_attribute>
-        <content_attribute id="violence-desecration">none</content_attribute>
-        <content_attribute id="violence-slavery">none</content_attribute>
-        <content_attribute id="violence-worship">none</content_attribute>
-        <content_attribute id="drugs-alcohol">none</content_attribute>
-        <content_attribute id="drugs-narcotics">none</content_attribute>
-        <content_attribute id="drugs-tobacco">none</content_attribute>
-        <content_attribute id="sex-nudity">none</content_attribute>
-        <content_attribute id="sex-themes">none</content_attribute>
-        <content_attribute id="sex-homosexuality">none</content_attribute>
-        <content_attribute id="sex-prostitution">none</content_attribute>
-        <content_attribute id="sex-adultery">none</content_attribute>
-        <content_attribute id="sex-appearance">none</content_attribute>
-        <content_attribute id="language-profanity">none</content_attribute>
-        <content_attribute id="language-humor">none</content_attribute>
-        <content_attribute id="language-discrimination">none</content_attribute>
-        <content_attribute id="social-chat">none</content_attribute>
-        <content_attribute id="social-info">none</content_attribute>
-        <content_attribute id="social-audio">none</content_attribute>
-        <content_attribute id="social-location">none</content_attribute>
-        <content_attribute id="social-contacts">none</content_attribute>
-        <content_attribute id="money-purchasing">none</content_attribute>
-        <content_attribute id="money-gambling">none</content_attribute>
-    </content_rating>
+    <translation type="gettext">com.usebottles.bottles</translation>
+    <content_rating type="oars-1.1"/>
     <url type="homepage">https://usebottles.com</url>
     <url type="bugtracker">https://github.com/bottlesdevs/Bottles/issues</url>
     <url type="help">https://docs.usebottles.com</url>
+    <url type="donation">https://usebottles.com/funding</url>
+    <url type="translate">https://hosted.weblate.org/engage/bottles</url>
+    <recommends>
+      <control>pointing</control>
+      <control>keyboard</control>
+      <control>touch</control>
+    </recommends>
+    <requires>
+      <display_length compare="ge">medium</display_length>
```

Comment:
```suggestion
      <display_length compare="ge">768</display_length>
```

@mirkobrombin I've read [this post](https://blogs.gnome.org/tbernard/2021/09/07/ready-for-software-41/) and as it points out, display_length size constants are broken, so it should be changed to its numerical equivalent for it to work properly. Sorry for the inconvenience 😅

Reply:
Oh no problem. Can you make another PR? If you can't, I'll fix it as soon as I can

Reply:
https://github.com/bottlesdevs/Bottles/pull/525 fixes it